tests/tracing/processor/test_mlflow_processor.py

```diff
@@ -42,6 +43,27 @@ def test_on_start(clear_singleton):
     assert child_span.attributes.get(SpanAttributeKey.REQUEST_ID) == json.dumps(_REQUEST_ID)
 
 
+def test_on_start_adjust_span_timestamp_to_exclude_backend_latency(clear_singleton):
+    trace_info = create_test_trace_info(_REQUEST_ID, 0)
+    mock_client = mock.MagicMock()
+
+    def _mock_start_tracked_trace():
+        time.sleep(0.5)  # Simulate backend latency
+        return trace_info
+
+    mock_client._start_tracked_trace.side_effect = _mock_start_tracked_trace
+    processor = MlflowSpanProcessor(span_exporter=mock.MagicMock(), client=mock_client)
+
+    original_start_time = time.time_ns()
+    span = create_mock_otel_span(trace_id=_TRACE_ID, span_id=1, start_time=original_start_time)
+
+    processor.on_start(span)
+
+    assert span.start_time != original_start_time
```

Comment:
> ?

Reply:
hehe it should be same in combination with the check below, but yea I can change it to be more explicit